### DISCUSSION TITLE
Actor Names - without requiring a list in the config file

### DISCUSSION THREAD

**Discussion by zwalk** [jnunemaker/flipper#865] [Ideas]
I like the concept of actor names, could be an opportunity to leave a quick note as to why an actor was added to a flipper, however, it seems like it would always require a code merge to update right now by adding it to the config file. What about the ability to just add a short description when adding the actor itself?

bonus idea: what about displaying the date the flipper was added?

**Comment by jnunemaker** [comment]
Hi! 

You don’t need a list in the config file. The actor name source is a block. It’s invoked on request so it can be dynamic. You can make it do model lookups or whatever you want to find the name. 

Check out the original PR for some ideas:
https://github.com/flippercloud/flipper/pull/737

If you need more examples let me know but hopefully that helps you get going.

**Reply by zwalk** [reply]
I think I was imagining a place in the Flipper::UI where you can do CRUD operations to the actor names source list, is that possible today?

**Reply by jnunemaker** [reply]
Gotcha. No nothing exists in the UI today for that. Cloud has that functionality but OSS does not. It's easier in cloud because we have one data store. To make that work in OSS we would have to expand adapters to store this or to be able to store arbitrary data which is a much bigger task.
